Assets/Scripts/GUI/LightingPopUpWindow.cs

```diff
@@ -46,6 +46,23 @@ override public void Init(GameObject rParent, string sText)
             //build list of lighting presets we're going to show
             m_Environments = EnvironmentCatalog.m_Instance.AllEnvironments.ToList();
 
+            // Remove passthrough scene for devices that don't support it
+            // (everything but Quest).
+            // TODO: Better way to detect Passthrough support.
+            // Extra: Passthrough should be *per* envrionment really!
+            // See https://github.com/icosa-foundation/open-brush/issues/456
+#if !OCULUS_SUPPORTED
```

Comment:
I'm nitpicking here, but can we do `#if OCULUS_SUPPORTED / #define PASSTHROUGH_SUPPORTED 1` and then replace this with `#if PASSTHROUGH_SUPPORTED`? It's just a little bit of technical debt that will be a pain to find later...